gto-support-db/src/main/java/org/ccci/gto/android/common/db/Expression.java

```diff
@@ -133,6 +133,46 @@ protected Binary binaryExpr(@NonNull final String op, @NonNull final Expression
     }
 
     @NonNull
+    public Aggregate count() {
```

Comment:
these aggregate functions should be moved to the `Field` class per the previous comment of narrowing Aggregate scope to `Field`s only